src/main/eo/org/eolang/txt/text.eo

```diff
@@ -99,9 +99,17 @@
       "".as-bytes
       [acc i x]
         if. > @
-          i.eq ((items.length).minus 1)
-          acc.concat (x.as-bytes)
-          (acc.concat (x.as-bytes)).concat s.as-bytes
+          eq.
```

Comment:
@EugeneDar To be honest, it looks less readable, as for me. 
What do you think about the next one?
```eo
        if. > @
          i.eq (items.length.minus 1)
          acc.concat x.as-bytes
          (acc.concat x.as-bytes).concat s.as-bytes
```

Reply:
@volodya-lombrozo sure, I'll rewrite it.